src/OpenTelemetry.Instrumentation.Hangfire/Implementation/HangfireInstrumentationJobFilterAttribute.cs

```diff
@@ -75,6 +83,11 @@ public void OnPerformed(PerformedContext performedContext)
                 activity.SetStatus(ActivityStatusCode.Error, performedContext.Exception.Message);
             }
 
+            if (performedContext.Exception != null && this.options.RecordException)
```

Comment:
nit: you can avoid checking Exception != null, if you move this into the existing if check for it above.

Reply:
yes, done it